deadfish.fbs

```diff
@@ -15,27 +15,31 @@ table Tileinfo {
 table Object {
     pos: Vec2;
     rotation: float;
+    size: Vec2;
     gid: uint16;
     hspotname: string;
 }
 
 table Decoration {
     pos: Vec2;
     rotation: float;
+    size: Vec2;
     gid: uint16;
 }
 
 table CollisionMask {
     pos: Vec2;
-    radius: float;
+    size: Vec2;
     rotation: float;
+    isCircle: bool;
     polyverts: [Vec2];
 }
 
 table HidingSpot {
     pos: Vec2;
-    radius: float;
+    size: Vec2;

```

Comment:
why is the size property introduced in a pr regarding tiles? is it needed for the tiles?